### PR TITLE
skipper-dev-to-alpha

### PR DESCRIPTION
**Pull request by muaazsaleem**
Cherry-picking https://github.com/zalando-incubator/kubernetes-on-aws/pull/1097 to alpha

### CODE REVIEW

File: cluster/manifests/kube-job-cleaner/cronjob.yaml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 metadata:
   name: kube-job-cleaner
   namespace: kube-system
   labels:
     application: kube-job-cleaner
-    version: "0.6"
+    version: "{{$version}}"
 spec:
-  schedule: "17 * * * *"
+  schedule: "*/5 * * * *"
```
Comment (szuecs) [inline cluster/manifests/kube-job-cleaner/cronjob.yaml]: this looks wrong to me

File: cluster/manifests/kube-proxy/daemonset.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: kube-system
   labels:
     application: kube-proxy
-    version: v1.9.7_coreos.0
+    version: v1.10.2_coreos.0
```
Comment (szuecs) [inline cluster/manifests/kube-proxy/daemonset.yaml]: this would rollout 1.10 before actual rolling out 1.10

File: cluster/master.clc.yaml [hunk 3]
```diff
@@ -126,7 +127,7 @@ systemd:
       After=docker.service dockercfg.service meta-data-iptables.service
 
       [Service]
-      Environment=KUBELET_IMAGE_TAG=v1.9.7_coreos.0
+      Environment=KUBELET_IMAGE_TAG=v1.10.2_coreos.0
```
Comment (szuecs) [inline cluster/master.clc.yaml]: this is the 1.10 rollout

**Comment by szuecs** [comment]
@muaazsaleem it's fine, we wait for the tests passing and then we can merge

**Comment by szuecs** [comment]
:+1:

**Comment by muaazsaleem** [comment]
👍

**Comment by linki** [comment]
👍

**Pull request merged**